### PR TITLE
Fev 419 configure npm scripts  contrib

### PR DESCRIPTION
**Pull request by alanovyy**

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @alanovyy,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: packages/playkit-js-scripts/scripts/contrib.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const contribTypes = {
+    LATEST: 'latest',
+    NEXT: 'next',
+    LOCAL: 'local',
+};
+
+const program = new commander.Command(packageJson.name)
+    .version(packageJson.version)
+    .option(
+        '--type <update-type>',
```
Comment (esakal) [inline packages/playkit-js-scripts/scripts/contrib.js]:
```suggestion
        '--type latest | next | local',
```

File: packages/playkit-js-scripts/scripts/contrib.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            Types could be:
+                - ${contribTypes.LATEST}
+                - ${contribTypes.NEXT}
+                - ${contribTypes.LOCAL}
+                
+            For example:> kcontrib contrib --type=local
+    `);
+    }
+
+    const appPath = process.cwd();
```
Comment (esakal) [inline packages/playkit-js-scripts/scripts/contrib.js]: this is new requirement, I will update jira as well:
1. show the following message to the user: `The following packages will be re-installed from npm with tag {latest|next}`
2. for local type message is 
```
The following packages will be linked to local libraries. 
if error happens you should run `npm run setup` in the contrib repository
```

image

File: packages/playkit-js-scripts/scripts/contrib.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    switch (program.type) {
+        case contribTypes.LATEST:
+            contribLink = '@latest';
+            break;
+        case contribTypes.NEXT:
+            contribLink = '@next';
+            break;
+        case contribTypes.LOCAL:
+            installStrategy = 'link';
+            contribLink = ' --production';
```
Comment (esakal) [inline packages/playkit-js-scripts/scripts/contrib.js]:
```suggestion
            contribLink = ' ';
```
it doesn't work with this provided flag

File: packages/playkit-js-scripts/scripts/contrib.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    let installStrategy = 'install';
+    let contribLink = null;
+
+
+    switch (program.type) {
+        case contribTypes.LATEST:
+            contribLink = '@latest';
+            break;
+        case contribTypes.NEXT:
+            contribLink = '@next';
```
Comment (esakal) [inline packages/playkit-js-scripts/scripts/contrib.js]: consider naming it to something more relevant `contribLink` is inccorect, maybe `npmTag`

File: packages/playkit-js-scripts/scripts/contrib.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    const packages = Object.keys(appPackage.dependencies)
+        .reduce((forInstall, dependency) => {
+            const [_, playkitPackage] = dependency.split(`${VARIABLES.CONTRIB}/`);
+
+            return playkitPackage ? [...forInstall, playkitPackage] : forInstall;
+        }, [])
+        .map(packageName => `${VARIABLES.CONTRIB}/${packageName}${npmTag}`);
+
+    console.log(
+        `The following packages will be linked to local libraries:\n`,
```
Comment (esakal) [inline packages/playkit-js-scripts/scripts/contrib.js]:
```suggestion
        `${INFO_MESSAGE}\n`,
```

**Pull request merged**